config/samples/example-test.sh

```diff
@@ -485,4 +485,16 @@ waitComponentPlanRetryTime "default" "do-once-nginx-sample-15.0.2" "5"
 info "5.7.4 verify this componentplan will failed, show error log"
 kubectl get cpl do-once-nginx-sample-15.0.2 --output='jsonpath={.status.conditions[?(@.type=="Actioned")]}'
 
+info "6 try to verify that the common steps are valid"
```

Comment:
maybe "try to verify that the common steps are valid to oci type"?

Reply:
The name is now changed to "6 try to verify that the common steps are valid to oci type"!